EESSI-pilot-install-software.sh

```diff
@@ -269,14 +269,34 @@ fail_msg="Installation of GROMACS failed, damned..."
 $EB GROMACS-2020.1-foss-2020a-Python-3.8.2.eb --robot
 check_exit_code $? "${ok_msg}" "${fail_msg}"
 
+# install custom CGAL without "'strict': True", which doesn't work on ppc64le
+export CGAL_EC="CGAL-4.14.3-gompi-2020a-Python-3.8.2.eb"
+echo ">> Installing custom CGAL easyconfig (${CGAL_EC})..."
+cd ${TMPDIR}
+#curl --silent -OL https://raw.githubusercontent.com/EESSI/software-layer/master/easyconfigs/${CGAL_EC}
+cp /swlayer/easyconfigs/${CGAL_EC} .
+cd - > /dev/null
+ok_msg="Custom CGAL installed!"
+fail_msg="Installation of CGAL failed, what the ..."
+$EB $TMPDIR/${CGAL_EC} --robot
+check_exit_code $? "${ok_msg}" "${fail_msg}"
+
 # note: compiling OpenFOAM is memory hungry (16GB is not enough with 8 cores)!
 # 32GB is sufficient to build with 16 cores
 echo ">> Installing OpenFOAM (twice!)..."
 ok_msg="OpenFOAM installed, now we're talking!"
 fail_msg="Installation of OpenFOAM failed, we were so close..."
-$EB OpenFOAM-8-foss-2020a.eb OpenFOAM-v2006-foss-2020a.eb --robot
+$EB --parallel=4 OpenFOAM-8-foss-2020a.eb OpenFOAM-v2006-foss-2020a.eb --robot --include-easyblocks-from-pr 2320
```

Comment:
The `parallel=4` part doesn't belong in here, should be set outside the script via `export EASYBUILD_PARALLEL=4`.
Is this need because of memory limitations?

Also, please add a comment to explain why the tweaked easyblock is needed

Reply:
Yes, was because of memory limitations. I've removed the line, though it's still a bit of a pity that you will have to run the entire script with that setting now.

Also added some more comments about this (and some other) customized easyblocks.